orchestration/src/main/java/com/sap/ai/sdk/orchestration/model/MultiChatMessageContent.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Orchestration
+ * Orchestration is an inference service which provides common additional capabilities for business AI scenarios, such as content filtering and data masking. At the core of the service is the LLM module which allows for an easy, harmonized access to the language models of gen AI hub. The service is designed to be modular and extensible, allowing for the addition of new modules in the future. Each module can be configured independently and at runtime, allowing for a high degree of flexibility in the orchestration of AI services.
+ *
+ * The version of the OpenAPI document: 0.43.0
+ *
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+package com.sap.ai.sdk.orchestration.model;
+
+import com.fasterxml.jackson.annotation.JsonSubTypes;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+
+/** MultiChatMessageContent */
+@JsonTypeInfo(use = JsonTypeInfo.Id.DEDUCTION)
+@JsonSubTypes({
+  @JsonSubTypes.Type(value = ImageContent.class),
+  @JsonSubTypes.Type(value = TextContent.class),
+})
+public interface MultiChatMessageContent {}
```

Comment:
(Minor)

I know changing generated code by hand is discouraged, but here we can make an exception:
Can you please mark this interface `@Beta`?